### PR TITLE
hotfix wasm themis: rebuild wasm-themis with -O3 flag

### PR DESCRIPTION
**Pull request by radetsky**
Previously I removed -O3 flag to make the code readable and ready to debug. I remember to return the flag but forgot to rebuild the package. Mea culpa. I return the flag and rebuild the package.

### CODE REVIEW

File: src/wrappers/themis/wasm/wasmthemis.mk [hunk 1]
```diff
@@ -39,9 +39,7 @@ $(BIN_PATH)/libthemis.js: LDFLAGS += -s LINKABLE=1
 # does not use are missing the from the binary. Suppress the warnings.
 $(BIN_PATH)/libthemis.js: LDFLAGS += -s ERROR_ON_UNDEFINED_SYMBOLS=0
 $(BIN_PATH)/libthemis.js: LDFLAGS += --pre-js $(WASM_PRE_JS)
-
 $(BIN_PATH)/libthemis.js: CMD = $(CC) -o $@ $(filter %.o %a, $^) -O3 $(LDFLAGS)
-
 $(BIN_PATH)/libthemis.js: $(THEMIS_STATIC) $(WASM_RUNTIME) $(WASM_PRE_JS)
```
Comment (vixentael) [inline src/wrappers/themis/wasm/wasmthemis.mk:43]: empty lines?

Comment (radetsky) [reply src/wrappers/themis/wasm/wasmthemis.mk:43]: It was empty lines.

Comment (vixentael) [reply src/wrappers/themis/wasm/wasmthemis.mk:43]: what is the point of this change? why it happened?

Screenshot 2022-08-22 at 20 29 38

Comment (radetsky) [reply src/wrappers/themis/wasm/wasmthemis.mk:43]: Cosmetic. I prefer to see one logical thing without new lines.

Comment (vixentael) [reply src/wrappers/themis/wasm/wasmthemis.mk:43]: removing empty lines is not a valuable change, it should not be included in this PR

**Review by vixentael — APPROVED** [review]

**Pull request merged**